starter-kits/nextjs-starter-kit/package.json

```diff
@@ -10,9 +10,9 @@
     "typecheck": "tsc --project tsconfig.json --noEmit"
   },
   "dependencies": {
-    "@liveblocks/client": "^0.19.10",
-    "@liveblocks/node": "^0.19.10",
-    "@liveblocks/react": "^0.19.10",
+    "@liveblocks/client": "^1.0.7",
+    "@liveblocks/node": "^1.0.7",
+    "@liveblocks/react": "^1.0.7",
```

Comment:
@CTNicholas Can you extra double-check this bump, please?

Reply:
@nvie Sorry, forgot about this earlier! You mean check it works on 1.0.7? Just gave it a try, all looks good 👍